db/migrate/20181024114437_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :admin, :boolean
```

Comment:
I think this should probably also default to false

Reply:
Yep that would be smart 😂